APE1.rst

```diff
@@ -88,11 +89,21 @@ doing this.
 Following a discussion on astropy-dev, the proposal should be submitted as a
 pull request to astropy-APEs with the name APE<n>.rst where <n> is an
 appropriately assigned number (i.e., not already an accepted/proposed/rejected APE).
-The draft must use the APEtemplate.rst file.
-That a formal proposal has been submitted as a PR should be announced to the
-astropy-dev list.
-
-The APE author may update the APE as needed.
+The draft must use the APEtemplate.rst file. Once the pull request is open,
+the ``@astropy/ape-editor-team`` should be mentioned in the pull request, and
```

Comment:
So, at merge time, this APE Editor Team should be in place already, no? Should setting up a team happen in parallel with this PR?

Or should CoCo take on this task until a proper team can be instated after?

Reply:
I don't think we can set up a team before it's merged, since it's not clear it's going to be accepted yet? I think it should just be set up if/when this PR is merged.